### PR TITLE
LGA-1212: Tech Debt - CSS tidy up and removal of references to old classes

### PR DESCRIPTION
**Pull request by MalcolmVonMoJ**
## What does this pull request do?

**Tech Debt**

- Deletes old CSS files that are no longer needed.  
- Removes styles that are obsolete.
- Removes references to unused classes.
- Removes unused JS functions that referenced obsolete classes.
- Style tweaks.
- Adopt GDS styles in a few minor overlooked places.
- Removes the GDS error colour bar (for 404 pages etc) that is no longer the norm.
- Amends the class and style of the character counter, to the GDS norm.  
- Amends JS for character counter.
- Unhides a hidden (with CSS) questions on contact page as per conversation with Josh and Alex.

## Any other changes that would benefit highlighting?

- Fixes a JS error that was affecting error messages on some questions.  

## Checklist

- [x] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

File: cla_public/translations/cy/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -1505,8 +1505,8 @@ msgid "Other language"
 msgstr "Iaith arall"
 
 #: cla_public/apps/contact/forms.py:40
-msgid "Language required:"
-msgstr "Iaith sydd ei hangen:"
+msgid "Language required"
+msgstr "Iaith sydd ei hangen"
```
Comment (mayowaadeniyi) [inline cla_public/translations/cy/LC_MESSAGES/messages.po:1509]: Are these lines still needed seeing as you removed them from the forms.py file?

File: cla_public/translations/en/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -1090,7 +1090,7 @@ msgid "Other language"
 msgstr ""
 
 #: cla_public/apps/contact/forms.py:40
-msgid "Language required:"
+msgid "Language required"
 msgstr ""
```
Comment (mayowaadeniyi) [inline cla_public/translations/en/LC_MESSAGES/messages.po:1094]: Are these lines still needed seeing as you removed them from the forms.py file?

File: cla_public/static-src/javascripts/modules/form-errors.js [hunk 3]
```diff
@@ -11,12 +11,6 @@
       $('form')
         .on('submit', this.postToFormErrors)
         // Focus back on summary if field-error is focused and escape key is pressed
-        .on('keyup', function(e) {
-          var $target = $(e.target);
-          if(e.keyCode === 27 && ($target.is('.govuk-form-group--error') || $target.parent().is('.govuk-form-group--error'))) {
-            $target.closest('form').find('> .alert').focus();
-          }
-        })
```
Comment (mayowaadeniyi) [inline cla_public/static-src/javascripts/modules/form-errors.js:19]: Do we want to replace the functionality or is it basically now redundant

File: cla_public/static-src/javascripts/modules/form-errors.js [hunk 4]
```diff
@@ -25,22 +19,6 @@
...
-          return;
-        }
-        var $target = $(targetId);
-
-        $('html, body').animate({
-          scrollTop: $target.offset().top - 20
-        }, 300, function() {
-          $target.attr('tabindex', -1).focus();
-        });
-      });
```
Comment (mayowaadeniyi) [inline cla_public/static-src/javascripts/modules/form-errors.js:43]: Do we want to replace the functionality or is it basically now redundant

File: cla_public/static-src/javascripts/modules/form-errors.js (hunk 3 above)
Comment (MalcolmVonMoJ) [reply cla_public/static-src/javascripts/modules/form-errors.js:19]: This is redundant.  Also it might clash with the escape key for quick exit.

File: cla_public/static-src/javascripts/modules/form-errors.js (hunk 4 above)
Comment (MalcolmVonMoJ) [reply cla_public/static-src/javascripts/modules/form-errors.js:43]: This has already been replaced as part of an earlier ticket.

File: cla_public/static-src/stylesheets/forms/_form-subfield.scss [hunk 5]
```diff
@@ -7,15 +7,6 @@
...
-    margin-bottom: 10px;
-
-    label:after {
-      content: none;
-    }
-  }
-
   .input-prefix {
     margin: 0 5px 0 0;
     display: inline-block;
```
Comment (mayowaadeniyi) [inline cla_public/static-src/stylesheets/forms/_form-subfield.scss:12]: Is this file still needed as it is removed from the forms SASS file

File: cla_public/static-src/stylesheets/_forms.scss [hunk 6]
```diff
@@ -1,21 +1,7 @@
-@import 'forms/inputs';
-@import 'forms/buttons';
 @import 'forms/field-help';
 @import 'forms/field-more-info';
-@import 'forms/field-error';
-@import 'forms/labels';
-@import 'forms/form-error';
 @import 'forms/form-group';
-@import 'forms/form-subfield';
```
Comment (mayowaadeniyi) [inline cla_public/static-src/stylesheets/_forms.scss:9]: Reference point for the other comment I made

File: cla_public/static-src/stylesheets/forms/_form-subfield.scss (hunk 5 above)
Comment (MalcolmVonMoJ) [reply cla_public/static-src/stylesheets/forms/_form-subfield.scss:12]: The fact that I deleted some of its content makes me think that the other content was still needed.  But there were so many, I can't remember.  I'll look into it.

File: cla_public/static-src/stylesheets/_forms.scss (hunk 6 above)
Comment (MalcolmVonMoJ) [reply cla_public/static-src/stylesheets/_forms.scss:9]: The fact that I deleted it here indicates that I don't think it is needed, but I can't remember so will double check.

Comment (MalcolmVonMoJ) [reply cla_public/static-src/stylesheets/_forms.scss:9]: This deletion was correct.

File: cla_public/static-src/stylesheets/forms/_form-subfield.scss (hunk 5 above)
Comment (MalcolmVonMoJ) [reply cla_public/static-src/stylesheets/forms/_form-subfield.scss]: This file should have been deleted.  Done now.

File: cla_public/static-src/stylesheets/_notice.scss [hunk 7]
```diff
@@ -1,9 +1,6 @@
+.get-in-touch,
 .notice {
-  border-top: 2px solid $grey-3;
```
Comment (mayowaadeniyi) [inline cla_public/static-src/stylesheets/_notice.scss:2]: Was deleting this line intentional?

Comment (MalcolmVonMoJ) [reply cla_public/static-src/stylesheets/_notice.scss:2]: Yes.  the border is now added in elsewhere (in the right colour).

**Review by mayowaadeniyi — APPROVED** [review]
LGTM 👍
